<?xml version="1.0"?>
<launch>
<param name ="/use_sim_time" value="true"/>

  <!-- <param name ="/use_sim_time" value="true"/> -->
  <arg name="play_bag" default="true" />
  <!-- <arg name="bag_file" default="/media/lzb/Windows-SSD/ubuntu16/datasets/uav_2019-11-21-19-33-27.bag"/> -->
  <arg name="bag_file" default="/media/lzb/Windows-SSD/ubuntu18/rosbag/jkw3_t265_l515/liu0.bag"/>

  <node name="player" pkg="rosbag" type="play" output="screen" args=" -r 1.0 --clock $(arg bag_file)" if="$(arg play_bag)" launch-prefix="xterm -e"/>
  <!-- <node pkg="tf" type="static_transform_publisher" name="cam_base_link" 
    args="0.0 0.0 0.0 0.0 0.785 0.0 t265_pose_frame d435i_link 100" /> -->

  <!-- <node pkg="tf" type="static_transform_publisher" name="cam_base_link1" 
    args="0.0 0.0 0.0 -1.57 0.0 0.0 d435i_link l515_link 100" /> -->

  <node name="tf2odom" pkg="ds_utils" type="tf2odom" output="screen"/>

<!--node pkg="rtabmap_ros" type="transform_to_tf.py" name="transform_to_tf">
  <remap from="transform" to="/t265/odom/sample"/>
  <param name="frame_id" value="t265_odom_frame"/>
  <param name="child_frame_id" value="t265_pose_frame"/>
</node-->


</launch>
